extensions/productboard/src/index.tsx

```diff
@@ -52,19 +52,54 @@ export default function Command() {
     }
   }
 
-  return (
+  const { FORM_FIELDS } = preferences;
+
+  if (!FORM_FIELDS) return (
     <Form
       actions={
         <ActionPanel>
           <SubmitFormAction title="Create Note" onSubmit={handleSubmit} />
         </ActionPanel>
       }
     >
-      <Form.TextField id="title" title="Title" placeholder="Enter title" />
-      <Form.TextArea id="content" title="Content" placeholder="Enter content of the note (markdown supported)" />
+      <Form.TextField id="title" title="Title" placeholder="Enter title..." />
+      <Form.TextArea id="content" title="Content" placeholder="Enter content of the note (markdown supported)..." />
+      <Form.Separator />
+      <Form.TextField id="tags" title="Tags" placeholder="Enter tags (separate by comma)..." defaultValue={preferences.TAGS_DEFAULT} />
+      <Form.TextField id="saidBy" title="Said by" placeholder="Enter valid email..." defaultValue={preferences.USER_EMAIL} />
+    </Form>
+  );
+
+  const fields = FORM_FIELDS.split(/,\s*/g);
+
+  async function handleFormTemplateSubmit(values: Omit<Note, 'content'> & FormValues) {
+    const { title, tags, saidBy } = values;
+    let content = '';
+    fields.forEach((field, index) => {
+      content += `<b>${field}:</b> ${values[index.toString()]} <br />`;
```

Comment:
Wouldn't it be better to format content here with markdown syntax, and let markdown converter in `handleSubmit` take care of the conversion to HTML? 

Also, have you tested markdown syntax in custom fields? Will it get converted properly?